src/main/java/com/google/sps/servlets/GameServlet.java

```diff
@@ -33,7 +36,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(jsonData);
 
+    Entity roundEntity = new Entity("Round");
+    roundEntity.setProperty("begin", System.currentTimeMillis() + 5000);
+    roundEntity.setProperty("end", System.currentTimeMillis() + 30000);
```

Comment:
for the 5000 and 30000, I would make these private static const with explanatory names. Others who look at the code might not understand what those values represent.